lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -94,4 +96,8 @@
   .nx-page-content {
     margin: 0 auto;
   }
+
+  .nx-page-main {
+    overflow: visible;
```

Comment:
Added because the dropdowns happened to be going beyond the edge of `.nx-page-main` and I discovered that they get cut off there.